stl/inc/deque

```diff
@@ -541,11 +535,18 @@ public:
     using const_reference = const value_type&;
     using _Mapptr         = typename _Val_types::_Mapptr;
 
+private:
+    static constexpr size_t _Bytes = sizeof(value_type);
+
+public:
+    static constexpr int _Block_size =
+        _Bytes <= 1 ? 16 : _Bytes <= 2 ? 8 : _Bytes <= 4 ? 4 : _Bytes <= 8 ? 2 : 1; // elements per block (a power of 2)

```

Comment:
Would it make sense to use something like 
```cpp
    static constexpr int _Block_size = (_STD min)(16 / static_cast<int>(_Bytes), 1);
```

Having written it, it does not look that much better though

Reply:
We might even reuse `ceil2`  from C++20

Reply:
We could also simply skip this as a rework of deque is intended for vNext

Reply:
By itself, `(_STD max)(16 / static_cast<int>(_Bytes), 1)` (not min) wouldn't work - `_Bytes == 5` would produce `3` which isn't a power of 2. So you would need the power-of-two function, and at that point there's no real clarity improvement. For vNext we may indeed need to use such a technique if the maximum block size is significantly larger.